upbound/platform-ref-azure
# feat(split): use abstractions for flux and prometheus

### Description of your changes
- bump uptest
- enable renovate
- remove service composition for flux, prometheus
- add configuration for xoss and gitops-flux (use abstractions)
- add yamllint
- bump providers


Fixes #

I have:


### How has this code been tested

```
--- PASS: kuttl (1587.36s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/case (1586.87s)
PASS
10:22:51 [ OK ] running automated tests
```
